connect/src/main/java/com/stripe/android/connect/webview/StripeConnectWebViewContainerController.kt

```diff
@@ -92,9 +125,31 @@ internal class StripeConnectWebViewContainerController<Listener : StripeEmbedded
         // don't send errors for requests that aren't for the main page load
         if (isMainPageLoad) {
             listener?.onLoadError(RuntimeException(errorString)) // TODO - wrap error better
+            analyticsService.track(
+                ConnectAnalyticsEvent.WebErrorPageLoad(
+                    status = httpStatusCode,
+                    error = errorMessage,
+                    url = requestUrl
+                )
+            )
         }
     }
 
+    fun onErrorDeserializingWebMessage(
+        webMessage: String,
+        error: String,
+        errorMessage: String?,
+    ) {
+        analyticsService.track(
+            ConnectAnalyticsEvent.WebErrorDeserializeMessage(
+                message = webMessage,
```

Comment:
Any security/privacy risks of sending this arbitrary data to analytics? We're sanitizing the URL for `component.web.error.unexpected_navigation` -- why isn't that necessary here?

Reply:
Yes, we do need to be sensitive of PII – please see my comment in the analytic event scoping doc:
https://docs.google.com/document/d/1nzHGofoclzij4nP5n5qgSpYLia-93heStXvt7vOfrkQ/edit?tab=t.0#bookmark=id.t7fuyluvgru2

When I originally scoped this, I checked that the JS messages we currently send shouldn't contain PII, but I'm considering removing it for future-proofing in case we eventually use a JS message that does contain PII.